<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeesDialog</class>
 <widget class="QDialog" name="EmployeesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Сотрудники</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/employees.ico</normaloff>:/icons/icons/employees.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="employeesTable">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>138</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Фамилия</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Отчество</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Должность</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Логин</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,2,1,2">
     <item>
      <widget class="QPushButton" name="addEmployeeButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="editEmployeeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="removeEmployeeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddEmployee">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/add2.ico</normaloff>:/icons/icons/add2.ico</iconset>
   </property>
   <property name="text">
    <string>Добавить сотрудника</string>
   </property>
   <property name="toolTip">
    <string>Добавить сотрудника</string>
   </property>
  </action>
  <action name="actionEditEmployee">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/edit2.ico</normaloff>:/icons/icons/edit2.ico</iconset>
   </property>
   <property name="text">
    <string>Изменить сотрудника</string>
   </property>
   <property name="toolTip">
    <string>Изменить данные сотрудника</string>
   </property>
  </action>
  <action name="actionRemoveEmployee">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/remove2.ico</normaloff>:/icons/icons/remove2.ico</iconset>
   </property>
   <property name="text">
    <string>Удалить сотрудника</string>
   </property>
   <property name="toolTip">
    <string>Удалить сотрудника</string>
   </property>
  </action>
  <action name="actionUpdateEmployees">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/update.ico</normaloff>:/icons/icons/update.ico</iconset>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="toolTip">
    <string>Обновить данные сотрудников</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
